### DISCUSSION TITLE
Update `prFooter` for GitHub hosted app

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#15739] [Suggest an Idea]
The PR footer text for the GitHub hosted app still uses an old link.

You should update the `prFooter` config option for the GitHub hosted app so it uses the `mend.io` domain.

```markdown
This PR has been generated by [Mend Renovate](https://renovate.whitesourcesoftware.com/). View repository job log [here](https://app.renovatebot.com/dashboard#github/renovatebot/renovate).
```

Check out PR https://github.com/renovatebot/renovate/pull/15738 for an example.

**Comment by rarkins** [comment]
Thanks, done
